examples/batched-processing/README.md

```diff
@@ -0,0 +1,39 @@
+# Batched Processing
+
+This example is an example of a minimal two-server aggregation scheme that
+fulfils the privacy guarantees of the Prio system.
+
+## Quickstart
+
+```bash
+# Build the containers
+make build
+
+# Start the server
+make test &
```

Comment:
Should be able to remove the `&` now, right?

Reply:
Yes, this is leftover from the previous method. There's no need to call `make clean` either.